### PR TITLE
Fixes #900 exporting pkml of a container should export all connected …

### LINKED ISSUE #900: Exporting PKML of a container => Export of neighborhoods
Exporting of a container to a PKML also exports all its sub-containers and all neighborhoods that have this container or its sub-containers as a neighbor.

### PR DESCRIPTION
**Pull request by msevestre**
…neighborhoods

### CODE REVIEW

File: tests/MoBi.Tests/Core/MoBiSpatialStructureSpecs.cs [hunk 1]
```diff
@@ -39,4 +47,64 @@ public void the_source_diagram_manager_creates_new_diagram_manager_for_target()
...
+            Name = "_randomNeighborhood"
+
+         };
+
+         sut.AddNeighborhood(_randomNeighborhood);
+      }
+
+      protected override void Because()
+      {
+         _result = sut.GetConnectingNeighborhoods(new[] {_parentContainer1, _container2, _container3}, _objectPathFactory);
```
Comment (msevestre) [inline tests/MoBi.Tests/Core/MoBiSpatialStructureSpecs.cs:101]: testing the behavior here that had not tests

File: tests/MoBi.Tests/Presentation/Tasks/EditTaskForContainerSpecs.cs [hunk 2]
```diff
@@ -76,10 +103,17 @@ protected override void Because()
...
-      public void the_entity_must_have_the_parent_container_restored_after_saving()
+      public void should_have_kept_the_reference_to_the_parent_container()
       {
          _entityToSave.ParentContainer.ShouldBeEqualTo(_parentContainer);
       }
+
+      [Observation]
+      public void should_have_exported_all_neighborhoods_connected_to_the_container_or_one_of_its_sub_containers()
+      {
+         _tmpSpatialStructure.Neighborhoods.ShouldContain(_neighborhood1);
```
Comment (msevestre) [inline tests/MoBi.Tests/Presentation/Tasks/EditTaskForContainerSpecs.cs:114]: also testing the export

File: src/MoBi.Core/Domain/Model/MoBiSpatialStructure.cs [hunk 3]
```diff
@@ -13,32 +13,25 @@ public class MoBiSpatialStructure : SpatialStructure, IWithDiagramFor<MoBiSpatia
       public IDiagramModel DiagramModel { get; set; }
       public IDiagramManager<MoBiSpatialStructure> DiagramManager { get; set; }
 
+      /// <summary>
+      ///    Returns all neighborhoods defined in the spatial structure having at least one neighbor connected to a neighbor
+      /// </summary>
+      /// <returns></returns>
```
Comment (rwmcintosh) [inline src/MoBi.Core/Domain/Model/MoBiSpatialStructure.cs:19]: Just remove that because you're already telling what's returned

Comment (rwmcintosh) [reply src/MoBi.Core/Domain/Model/MoBiSpatialStructure.cs:19]: <returns></returns>

**Review by rwmcintosh — APPROVED** [review]

**Pull request merged**